tests/framework/microvm.py

```diff
@@ -926,6 +926,21 @@ def ssh(self):
         """Return a cached SSH connection on the 1st interface"""
         return self.ssh_iface(0)
 
+    def verify_if_guest_is_up(self, up=True):
+        """Verify if the guest running inside the microVM is up and running.
+
+        It is checked by running a command via SSH connection. This method can also be used to wait
+        for guest to boot up.
+
+        :param up: expected to be up (True) or down (False).
+        """
+        rc, _, stderr = self.ssh.run("true")
+        if up:
```

Comment:
hmm since only one test needs this, we could just have that test do:

```python
with pytest.raises(AssertionError):
    uv.verify_if_guest_is_up()
```

Reply:
you're right! will apply your suggestion.